syft/cataloger/python/parse_wheel_egg_metadata.go

```diff
@@ -71,8 +70,23 @@ func parseWheelOrEggMetadata(path string, reader io.Reader) (pkg.PythonPackageMe
 
 	// add additional metadata not stored in the egg/wheel metadata file
 
-	sitePackagesRoot := filepath.Clean(filepath.Join(filepath.Dir(path), ".."))
-	metadata.SitePackagesRootPath = sitePackagesRoot
+	metadata.SitePackagesRootPath = filepath.Clean(filepath.Join(filepath.Dir(path), ".."))
 
 	return metadata, nil
 }
+
+// handleFieldBodyContinuation returns the updated value for the specified field after processing the specified line.
+// If the continuation cannot be processed, it returns an error.
+func handleFieldBodyContinuation(key, line string, fields map[string]string) (string, error) {
```

Comment:
I take it the linter rules nudged this above the function length due to added comments?...

Reply:
@wagoodman Yep, exactly — see the PR description

Reply:
if only there was a spot in a PR where I could read about context... 🤔 ...GREAT SCOTT!! When did they add PR descriptions??!! (...my bad)

Reply:
![great scott](http://3.bp.blogspot.com/-2wYEMgaLmnY/UTkehjyDQXI/AAAAAAAAGT4/ggODYP7m054/s1600/Christopher%2BLloyd%2BGreat%2BScott%2Bgif.gif)